app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+
+  validates :title, presence: true, uniqueness: true
+
+  def self.media_spotlight
+    all_works = self.all
+    if all_works.length > 0
+      popular_works = all_works.sort_by { |work| -work.votes.count }
+      return popular_works.first
+    else
+      return nil
+    end
+  end
```

Comment:
Refactoring thought: Actually, you don't need the `if ... else`. Your method can just look like this:

```ruby
  def self.media_spotlight
    popular_works = self.all.sort_by { |work| -work.votes.count }
    return popular_works.first
  end
```

Food for thought: Why does this work? Consider what this line returns in irb: `[].first`